### PR TITLE
chore: add a code of conduct and a contributing guide

### LINKED ISSUE #119: Community Profile
This repository follows very strict committing and contributing guidelines. It would be very beneficial to the maintainers of this project if we laid out exactly what we want and expect of our contributors in a `CONTRIBUTING.md` file. I could try to write one up later, but I think it'd be best if, @oupala, you could write it, as you facilitate the activities this project.

Also, would it be beneficial to add both issue and pull request templates? I found some really nice ones [here](https://github.com/stevemao/github-issue-templates/tree/master/simple) that I really like.

And, while I'm on the topic, how about a code of conduct? I'd say it's not necessary, but would be kind of nice to have. 

All of the above things I've mentioned can be found on [this page](https://github.com/oupala/apaxy/community).

### PR DESCRIPTION
**Pull request by oupala**
The code of conduct has been generated using the `covgen` package.

Email address  has to be updated in order to use a valid email address.

Fixes #119

## Proposed Changes

- add a code of conduct

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at mail@example.org. All
```
Comment (jordanbancino) [inline CODE_OF_CONDUCT.md]: Should we put an actual address here?

Comment (oupala) [reply CODE_OF_CONDUCT.md]: Yes, we should replace the dummy email address by a real one. Does anyone knows a free and easy email redirector? Or should we use an email distribution list?

**Comment by oupala** [comment]
I've just added a contributing guide. Please tell what you think of it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+### Closing merge requests
+
+The original poster or the maintainer's of apaxy may close a merge request at
+any time. Typically, but not exclusively, issues are closed when:
+
+- The merge request has conflict with the develop branch that makes it difficult
+to merge without a hard work
+- The commit message of each commit of the merge request does not follow the
+[conventional commits](https://www.conventionalcommits.org/) standard
```
[the remark this answers is not in the document]
Comment (oupala) [reply CONTRIBUTING.md]: I think that the most confusing thing was the conventional commits thing, not the conflict thing. Am I right?

[the remark this answers is not in the document]
Comment (jordanbancino) [reply CONTRIBUTING.md]: Yes, conventional commits was the intended target of that comment.

[the remark this answers is not in the document]
Comment (oupala) [reply CONTRIBUTING.md]: This is useful to generate a changelog automatically from the commits messages.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (oupala) [reply CODE_OF_CONDUCT.md]: I ended using a free email redirector service.

This make it easier to change the redirection email, but we'll have to store the service account credentials.

**Comment by oupala** [comment]
Pull request is ready for merge.

@jordanbancino can you please tell me if you think everything is ok or not?

**Comment by oupala** [comment]
Well, as it seems that we are the only currently active people on apaxy these days, I will merge the pull request.

**Pull request merged**